<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MNIST</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Draw from</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QPushButton" name="btnDraw">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>2</x>
              <y>62</y>
              <width>121</width>
              <height>25</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Draw</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="leStart">
            <property name="geometry">
             <rect>
              <x>2</x>
              <y>31</y>
              <width>121</width>
              <height>25</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDraw">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>112</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnLoadMnist">
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTraining">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Training</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoadMnist</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>loadMnistData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTraining</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>training()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDraw</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>drawImageOnTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadMnistData()</slot>
  <slot>drawImageOnTable()</slot>
  <slot>training()</slot>
 </slots>
</ui>
